src/commands/skip.rs

```diff
@@ -20,9 +23,43 @@ pub async fn skip(
 
     if queue.is_empty() {
         return create_response(&ctx.http, interaction, NOTHING_IS_PLAYING).await;
-    } else if queue.skip().is_ok() {
-        return create_response(&ctx.http, interaction, SKIPPED).await;
     }
 
+    let mut data = ctx.data.write().await;
+    let cache_map = data.get_mut::<GuildCacheMap>().unwrap();
+
+    let cache = cache_map.entry(guild_id).or_default();
+
+    if !cache.current_skip_votes.contains(&interaction.user.id) {
+        cache.current_skip_votes.push(interaction.user.id);
+    }
```

Comment:
No longer needs this check if `cache.current_skip_votes` is a HashSet instead of a vector.